diff --git a/Configure b/Configure
index d02f92d..bcb6755 100755
--- a/Configure
+++ b/Configure
@@ -352,15 +352,15 @@ echo "	\$(RM) -f config.h Makefile" >> $FILE
 echo "install-arch: build-arch" >> $FILE
 echo "	install -d \$(DESTDIR)$PREFIX/bin" >> $FILE
 echo "	install -d \$(DESTDIR)$PREFIX/share/man/man1" >> $FILE
-echo "	install -t \$(DESTDIR)$PREFIX/bin sympow" >> $FILE
-echo "	install -t \$(DESTDIR)$PREFIX/share/man/man1 sympow.1" >> $FILE
+echo "	install sympow \$(DESTDIR)$PREFIX/bin" >> $FILE
+echo "	install sympow.1 \$(DESTDIR)$PREFIX/share/man/man1" >> $FILE
 echo "install-indep: build-indep" >> $FILE
 echo "	install -d \$(DESTDIR)$PREFIX/lib/sympow" >> $FILE
 echo "	install -d \$(DESTDIR)$PREFIX/share/sympow" >> $FILE
 echo "	install -d \$(DESTDIR)$PREFIX/share/sympow/datafiles" >> $FILE
-echo "	install -t \$(DESTDIR)$PREFIX/lib/sympow new_data" >> $FILE
-echo "	cp -t \$(DESTDIR)$PREFIX/share/sympow standard1.gp standard2.gp standard3.gp" >> $FILE
-echo "	cp -t \$(DESTDIR)$PREFIX/share/sympow/datafiles datafiles/*.txt datafiles/param_data" >> $FILE
+echo "	install new_data \$(DESTDIR)$PREFIX/lib/sympow" >> $FILE
+echo "	cp standard1.gp standard2.gp standard3.gp \$(DESTDIR)$PREFIX/share/sympow" >> $FILE
+echo "	cp datafiles/*.txt datafiles/param_data \$(DESTDIR)$PREFIX/share/sympow/datafiles" >> $FILE
 echo "install: install-arch install-indep" >> $FILE
 ####if [ -n "$TAR" ];
 ####then
